atccmd/command.go

```diff
@@ -106,6 +106,7 @@ type ATCCommand struct {
 	ResourceCheckingInterval          time.Duration `long:"resource-checking-interval" default:"1m" description:"Interval on which to check for new versions of resources."`
 	OldResourceGracePeriod            time.Duration `long:"old-resource-grace-period" default:"5m" description:"How long to cache the result of a get step after a newer version of the resource is found."`
 	ResourceCacheCleanupInterval      time.Duration `long:"resource-cache-cleanup-interval" default:"30s" description:"Interval on which to cleanup old caches of resources."`
+	WorkerSchedulingMethod            string        `long:"worker-scheduling-method" default:"bucket" choice:"bucket" choice:"random" description:"How jobs shoudld be scheduled across the worker pool."`
```

Comment:
Could we rename this to `ContainerPlacementStrategy`, rename `bucket` to `volume-locality`, and have the description say "Method by which a worker is selected during container placement." or something?

I think that more precisely captures what it does.

Reply:
Sounds good, updated!